src/lib/stitching/vortex/stitching.ts

```diff
@@ -318,17 +322,27 @@ export const vortexStitchingEnvironment = (localSchema: GraphQLSchema) => ({
               ... on AnalyticsArtist {
                 entityId
               }
+              ... on AnalyticsViewingRoom {
+                entityId
+              }
             }
           }
         `,
         resolve: (parent, _args, context, info) => {
           const removeVortexPrefix = (name) => name.replace("Analytics", "")
           const typename = parent.rankedEntity.__typename
-          const fieldName = removeVortexPrefix(typename).toLowerCase()
+          const typenameWithoutPrefix = removeVortexPrefix(typename)
+          // Data massaging to allow for proper casing for query: viewingRoom, show, artwork, etc
+          const fieldName =
+            typenameWithoutPrefix.charAt(0).toLowerCase() +
+            typenameWithoutPrefix.slice(1)
```

Comment:
kinda makes you wish strings were mutable in js huh? well, this looks like it's the best way to me!